Content.Shared/Administration/Components/SharedHeadstandComponent.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Content.Shared.Administration.Components;
+
+public abstract class SharedHeadstandComponent : Component
+{
+
+}
```

Comment:
Comments. I assume it turns them upside down but I would assume that would be a handstand.